controller/space_iterations_test.go

```diff
@@ -541,31 +536,31 @@ func (rest *TestSpaceIterationREST) createIterations() (spaceID uuid.UUID, fathe
 				SpaceID:    spaceID,
 				StartAt:    &start,
 				EndAt:      &end,
-				UserActive: &userActive,
+				UserActive: false,
 			}
 			repo.Create(rest.ctx, &i)
 		}
 		// create one child iteration and test for relationships.Parent
 		fatherIteration = &iteration.Iteration{
 			Name:       "Parent Iteration",
 			SpaceID:    spaceID,
-			UserActive: &userActive,
+			UserActive: false,
 		}
 		repo.Create(rest.ctx, fatherIteration)
 		rest.T().Log("fatherIteration:", fatherIteration.ID, fatherIteration.Name, fatherIteration.Path)
 		childIteration = &iteration.Iteration{
 			Name:       "Child Iteration",
 			SpaceID:    spaceID,
 			Path:       append(fatherIteration.Path, fatherIteration.ID),
-			UserActive: &userActive,
+			UserActive: false,
 		}
 		repo.Create(rest.ctx, childIteration)
 		rest.T().Log("childIteration:", childIteration.ID, childIteration.Name, childIteration.Path)
 		grandChildIteration = &iteration.Iteration{
 			Name:       "Grand Child Iteration",
 			SpaceID:    spaceID,
 			Path:       append(childIteration.Path, childIteration.ID),
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c